app/src/main/java/com/vero/tasky/agenda/presentation/agenda/AgendaViewModel.kt

```diff
@@ -71,6 +72,13 @@ class AgendaViewModel @Inject constructor(
                     )
                 }
             }
+            AgendaEvent.RefreshAgenda -> {
+                viewModelScope.launch {
+                    updateState(newState = state.copy(isRefreshing = true))
```

Comment:
The updateState method was used to save the state in case of process death, right?

Personally, I don't save the loading state to survive process death
Since if the user leaves to process death just before line 79, then they will be stuck on a loading screen